src/schema/v2/HeroUnit/HeroUnitType.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { InternalIDFields } from "../object_identification"
+import { ResolverContext } from "types/graphql"
+import Image, { normalizeImageData } from "../image"
+
+interface HeroUnitTypeProps {
+  body: string
+  created_at: string
+  credit?: string
+  end_at?: string
+  image?: {
+    image_url?: string
+    image_urls?: string[]
+    image_versions?: string[]
+    original_height?: string
+    original_width?: string
+  }
+  label?: string
+  link_text: string
+  link_url: string
+  position?: number
+  start_at?: string
+  title: string
+}
+
+export const HeroUnitType = new GraphQLObjectType<
+  HeroUnitTypeProps,
+  ResolverContext
+>({
+  name: "HeroUnit",
+  description: "A Hero Unit",
+  fields: {
+    ...InternalIDFields,
+    body: {
+      description: "Main Hero Unit content.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    createdAt: {
+      resolve: ({ created_at }) => created_at,
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    credit: {
+      description: "Optional image credit line.",
+      type: GraphQLString,
+    },
+    endAt: {
+      description:
+        "Timestamp for when the Hero Unit should no longer be running.",
+      resolve: ({ end_at }) => end_at,
+      type: GraphQLString,
+    },
+    image: {
+      description: "The main image for the Hero Unit.",
+      type: Image.type,
+      resolve: ({ image }) => {
+        if (!image) return null
+        return normalizeImageData(image)
+      },
+    },
+    label: {
+      description: "Optional label for above the title.",
+      type: GraphQLString,
+    },
+    linkText: {
+      description: "Text for the CTA of the Hero Unit.",
+      resolve: ({ link_text }) => link_text,
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    linkUrl: {
+      description: "URL for the CTA of the Hero Unit.",
+      resolve: ({ link_url }) => link_url,
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    position: {
+      description: "Dictates the order of the Hero Units.",
+      type: GraphQLInt,
+    },
+    startAt: {
+      description: "Timestamp for when the Hero Unit should begin running.",
+      resolve: ({ start_at }) => start_at,
```

Comment:
We have a `date` field type that will give us formatting options.

Reply:
Ok I updated the 3 date fields to use that `date` type instead! 🙌